website/src/pages/api/chat/assistant_message.ts

```diff
@@ -1,8 +1,12 @@
+import { boolean } from "boolean";
 import { withoutRole } from "src/lib/auth";
 import { createInferenceClient } from "src/lib/oasst_inference_client";
 import { InferencePostAssistantMessageParams } from "src/types/Chat";
 
 const handler = withoutRole("banned", async (req, res, token) => {
+  if (!boolean(process.env.ENABLE_CHAT)) {
```

Comment:
this line is repeated too many times, maybe we can re-add the `isChatEnabled` but we can use the new `getEnv()` in it

Reply:
Done, I dont still dont like the inChatEnable solution, but it's simple for now. The perfect solution should be using some data transformer lib like `zod` or something